### PR TITLE
Cleaned up README.md

### PR DESCRIPTION
**Pull request by tgreve**
I deleted the content of the old README.md file.
Currently, the file simply says:
This repository contains a few useful functions.

### CODE REVIEW

**Comment by florpi** [comment]
Beautiful

File: Greve/template/README.md [hunk 1]
```diff
@@ -1,13 +1 @@
...
-    >>> factorial(10)
-    9
-
-and this other part does something.  I forget why that I did it:
-
-    >>> fibonnaccci(100)
-    [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
-
-If you can't use it, its kind of your problem, not mine!
+This repository contains a few useful functions.
```
Comment (ignotur) [inline Greve/template/README.md]: Maybe it is better to list these functions and describe their purpose and arguments.

Comment (tgreve) [reply Greve/template/README.md]: You are right. I was being lazy. Let me do that now and make these changes to my pull request. Cheers - T

Comment (tgreve) [reply Greve/template/README.md]: Happy now? :-)

**Comment by eteq** [comment]
Thanks for opening the pull request and for the review(s)!  We will merge this shortly.  Below I have provided a few comments about the *process* of the pull request and review. But note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code.

@ignotur - the review was good here, but one minor thing: you did not "approve" or "request changes" when you submitted the review.  Without that a maintainer isn't sure if your review should be taken as a complete review or just general comments.

@florpi - your comment, while positive and encouraging, did not provide any specific feedback for this PR. If you instead meant to indicate that you had read through all the code and were happy with it, you could have used the "approve" to indicate that.

**Pull request merged**